test/integration/testutils/utils.go

```diff
@@ -33,6 +35,33 @@ func FileExists(filePath string) (bool, error) {
 	return false, err
 }
 
+func DisableAutoTfVarsFile(rootDir string) error {
```

Comment:
Is `DisableAutoTfVarsFile()` currently used?  I'd like to avoid adding it if possible.

Reply:
Otherwise this PR looks good to me.

Reply:
The function is not being used anymore, I removed it now